README.md

```diff
@@ -4,18 +4,21 @@
 [![Build](https://github.com/temporalio/sdk-ruby/actions/workflows/build-native-ext.yml/badge.svg)](https://github.com/temporalio/sdk-ruby/actions/workflows/build-native-ext.yml?query=branch%3Amain+event%3Arelease)
 [![Gem Version](https://badge.fury.io/rb/temporalio.svg)](https://rubygems.org/gems/temporalio)
 
+⚠️ INACTIVE ⚠️ 
+
+This repository is not under active development, is client only, and is not compatible with Ruby 3+.
```

Comment:
I imagine reading this, being curious if it will be active in the future. If we don't know the answer, even a vague one, maybe we could at least say: "we plan on continuing development at some point in the future once we have bandwidth for it, but do not have a timeline."

Reply:
Thanks for reviewing! The purpose here is to negate ambiguity, so purposely not introducing any language like that.